README.md

```diff
@@ -1,40 +1,35 @@
+[![Travis](https://img.shields.io/travis/cedarcode/webauthn-ruby/master.svg?style=flat-square)](https://travis-ci.org/cedarcode/webauthn-ruby)
+[![Join the chat at https://gitter.im/cedarcode/webauthn-ruby](https://badges.gitter.im/cedarcode/webauthn-ruby.svg)](https://gitter.im/cedarcode/webauthn-ruby?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
 ## What is WebAuthn?
 
-- [WebAuthn W3C Recommendation](https://www.w3.org/TR/webauthn/)
-- [WebAuthn intro by Yubico](https://www.yubico.com/webauthn/)
-- [WebAuthn in Wikipedia](https://en.wikipedia.org/wiki/WebAuthn)
-- [Web Authentication API in MDN](https://developer.mozilla.org/en-US/docs/Web/API/Web_Authentication_API)
-- [WebAuthn article with Google IO 2018 talk](https://developers.google.com/web/updates/2018/05/webauthn)
+WebAuthn (Web Authentication) is a W3C standard for secure public-key authentication on the Web supported by all leading browsers and platforms.
 
-## Prerequisites
+For more:
+
+- WebAuthn [W3C Recommendation](https://www.w3.org/TR/webauthn/) (i.e. "The Standard")
+- WebAuthn [intro](https://www.yubico.com/webauthn/) by Yubico
+- WebAuthn [article](https://en.wikipedia.org/wiki/WebAuthn) in Wikipedia
+- [Web Authentication API](https://developer.mozilla.org/en-US/docs/Web/API/Web_Authentication_API) in MDN
+- WebAuthn [article with talk](https://developers.google.com/web/updates/2018/05/webauthn) in Google Developers
 
-This gem will help your ruby server act as a conforming [_Relying-Party_](https://www.w3.org/TR/webauthn/#relying-party), in WebAuthn terminology. But for the [_Registration_](https://www.w3.org/TR/webauthn/#registration) and [_Authentication_](https://www.w3.org/TR/webauthn/#authentication) ceremonies to work, you will also need
+## Prerequisites
 
-### A conforming User Agent
+This ruby library will help your Ruby/Rails server act as a conforming [_Relying-Party_](https://www.w3.org/TR/webauthn/#relying-party), in WebAuthn terminology. But for the [_Registration_](https://www.w3.org/TR/webauthn/#registration) and [_Authentication_](https://www.w3.org/TR/webauthn/#authentication) ceremonies to fully work, you will also need to add two more pieces to the puzzle, a conforming [User Agent](https://www.w3.org/TR/webauthn/#conforming-user-agents) + [Authenticator](https://www.w3.org/TR/webauthn/#conforming-authenticators) pair.
 
-Currently supporting [Web Authentication API](https://developer.mozilla.org/en-US/docs/Web/API/Web_Authentication_API):
-  - [Mozilla Firefox](https://www.mozilla.org/firefox/) 60+
-  - [Google Chrome](https://www.google.com/chrome/) 67+
-  - [Google Chrome for Android](https://play.google.com/store/apps/details?id=com.android.chrome) 70+
+Examples of known conformant pairs are for example:
 
-### A conforming Authenticator
+- Google Chrome for Android 70+ and Android's Fingerprint-based platform authenticator
```

Comment:
```suggestion
- Google Chrome for Android 70+ and Android's platform authenticator
```
Any unlock method should work, e.g. screen pattern :)

Reply:
Yes, that's true.

The thing is I wanted to provide clarity to the reader by providing real-world specific examples, instead of being generic and more "correct".